binding/java/src/ai/picovoice/rhino/Rhino.java

```diff
@@ -52,8 +52,8 @@
      * @param sensitivity Inference sensitivity. It should be a number within [0, 1]. A higher
      *                    sensitivity value results in fewer misses at the cost of (potentially)
      *                    increasing the erroneous inference rate.
-     * @param requireEndpoint If set to `true`, Rhino requires an endpoint (chunk of silence) before finishing inference.
-     * @throws RhinoException if there is an error while initializing Rhino.
+     * @param requireEndpoint If set, Rhino requires an endpoint (chunk of silence) before finishing inference.
```

Comment:
sorry, I meant to just change the usage description in the command-line demo usage. The description can still say `If set to 'true'..`

Reply:
my bad!